docs/UserGuide.md

```diff
@@ -43,33 +43,51 @@ simpler to use while being even more functional than alternatives on the market.
 
 ## Quick start
 
-1. Ensure you have Java `17` or above installed in your Computer. If not, you can download Java `17` [here](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html).
+### Step 1: Check if Your Computer is Ready
+First, we need to make sure your computer has Java 17 installed. Here's how to check:
 
-2. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-F15-3/tp/releases).
+1. Open your computer's terminal:
+   - **For Windows**: Press the Windows key + R, type `cmd`, and press Enter
+   - **For Mac**: Press Command + Space, type `terminal`, and press Enter
+2. In the black window that appears, type exactly:
+   ```
+   java --version
+   ```
+   and press Enter
+3. What you should see:
+   - ✅ If you see "Java version 17" (or any number above 17), you're ready to go!
```

Comment:
I think this might be able to be rephrased better? For example when I run the command, `Java version 17` does not appear word for word. Or maybe its an apple thing (?)

![image](https://github.com/user-attachments/assets/eb8a2992-c0ef-45d4-b557-840a02befcf6)

Reply:
I will use java 17 or openjdk 17 then